src/StackAdmin/Azs.Compute.Admin/Module/Azs.Compute.Admin/Generated.PowerShell.Commands/SwaggerPathCommands/Set-AzsComputeQuota.ps1

```diff
@@ -69,6 +75,14 @@ function Set-AzsComputeQuota {
         [int32]
         $VirtualMachineCount,
 
+        [Parameter(Mandatory = $false)]
+        [int32]
+		$MaxAllocationStandardManagedDisksAndSnapshots,
+
+        [Parameter(Mandatory = $false)]
+        [int32]
+		$MaxAllocationPremiumManagedDisksAndSnapshots,
```

Comment:
Same as above.

Reply:
This has been discussed and PM has opened a bug to track this and get done.

Reply:
We will just have to make a breaking change in the future and notify clients with a warning before we do.